tests/tracer/test_propagation.py

```diff
@@ -185,6 +217,48 @@ def test_extract(tracer):
             }
 
 
+@pytest.mark.subprocess(
+    env=dict(DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED="true"),
+    parametrize={
+        "TEST_TRACE_ID": [
+            str(2 ** 128 - 1),
+            str(2 ** 127 + 1),
+            str(2 ** 65 - 1),
+            str(2 ** 64 + 1),
```

Comment:
`2**64` is an interesting case. Here the higher order bits will be `0000000000000001` and the trace_id_64bit will be zero. trace_id_64bit can not be zero, this will break `x-datadog-trace-id` headers in older versions of the tracer.

Reply:
That's why I used `2 ** 64 + 1` instead. Here the higher order bits will be set to `0000000000000001` and the trace_id_64bit will be `1`